src/components/Element/Note.jsx

```diff
@@ -1,44 +1,66 @@
 import Icon from './icon';
+import { useEffect, useState } from 'react';
 
-function Button({ children }) {
+function Button({ children, onClick }) {
   return (
-    <button className="text-gray-600 transition-colors duration-500 hover:text-black">
+    <button
+      className="text-gray-600 transition-colors duration-500 hover:text-black"
+      onClick={onClick}
+    >
       {children}
     </button>
   );
 }
 
 const ButtonList = ['Tierney Bricker', 'Eonline.com'];
 
-function NoteList() {
+const Vocabulary = {
+  index: 1,
+  title: 'Latest',
+  translation: '最新的、新鮮的、新鮮、最新',
+};
+
+function Task({ vocabulary }) {
   return (
-    <ul className="mt-10">
-      <li className="flex justify-between">
-        <ul className="flex space-x-4">
-          <li>
-            <span>1.</span>
-            <span>Latest</span>
-          </li>
-          <li>最新的、新鮮的、新鮮、最新</li>
-        </ul>
-        <div className="flex space-x-4">
-          <button className="relative flex border border-black p-2">
-            <span className="w-5">
-              <Icon.Edit />
-            </span>
-          </button>
-          <button className="relative flex border border-black p-2">
-            <span className="w-4">
-              <Icon.Delete />
-            </span>
-          </button>
-        </div>
-      </li>
-    </ul>
+    <li className="flex justify-between py-2">
+      <ul className="flex space-x-4">
+        <li>
+          <span>{vocabulary.index}.</span>
+          <span>{vocabulary.title}</span>
+        </li>
+        <li>{vocabulary.translation}</li>
+      </ul>
+      <div className="flex space-x-4">
+        <button className="relative flex border border-black p-2">
+          <span className="w-5">
+            <Icon.Edit />
+          </span>
+        </button>
+        <button className="relative flex border border-black p-2">
+          <span className="w-4">
+            <Icon.Delete />
+          </span>
+        </button>
+      </div>
+    </li>
   );
 }
 
+function NoteList({ children }) {
+  return <ul className="mt-10">{children}</ul>;
+}
+
 export function Note() {
+  const [list, setList] = useState([]);
+
+  function addItem() {
+    setList((list) => list.concat(Vocabulary));
```

Comment:
This works right now because you have a vocabulary object to concat, but this function should accept a parameter to concat into the list right?